arches_querysets/datatypes/concept_types.py

```diff
@@ -0,0 +1,88 @@
+import uuid
+
+from arches.app.datatypes import concept_types
+from arches.app.utils.betterJSONSerializer import JSONDeserializer, JSONSerializer
+
+
+class ConceptDataType(concept_types.ConceptDataType):
+    def transform_value_for_tile(self, value, **kwargs):
+        if isinstance(value, dict) and (value_id := value.get("valueid")):
+            return super().transform_value_for_tile(value_id)
+        return super().transform_value_for_tile(value)
+
+    def to_python(self, value, **kwargs):
+        return self.get_instance(value)
+
+    def to_json(self, tile, node):
+        return {
+            "@display_value": self.get_display_value(tile, node),
+            "@details": self.get_details(tile, node),
+        }
+
+    def get_details(self, tile, node):
```

Comment:
could `get_details` get folded into another function, or renamed? It's a bit vague currently